detection_rules/devtools.py

```diff
@@ -210,6 +219,83 @@ def license_check(ctx):
     ctx.exit(int(failed))
 
 
+@dev_group.command('package-stats')
+@click.option('--token', '-t', help='GitHub token to search API authenticated (may exceed threshold without auth)')
+@click.option('--threads', default=50, help='Number of threads to download rules from GitHub')
+@click.pass_context
+def package_stats(ctx, token, threads):
+    """Get statistics for current rule package."""
+    current_package: Package = ctx.invoke(build_release, verbose=False, release=False)
+    release = f'v{current_package.name}.0'
+    new, modified, errors = rule_loader.load_gh_pr_rules(labels=[release], token=token, threads=threads)
+
+    click.echo(f'Total rules as of {release} package: {len(current_package.rules)}')
+    click.echo(f'New rules: {len(current_package.new_rules_ids)}')
+    click.echo(f'Modified rules: {len(current_package.changed_rule_ids)}')
+    click.echo(f'Deprecated rules: {len(current_package.removed_rule_ids)}')
+
+    click.echo('\n-----\n')
+    click.echo('Rules in active PRs for current package: ')
+    click.echo(f'New rules: {len(new)}')
+    click.echo(f'Modified rules: {len(modified)}')
+
+
+@dev_group.command('search-rule-prs')
+@click.argument('query', required=False)
+@click.option('--no-loop', '-n', is_flag=True, help='Run once with no loop')
```

Comment:
how about we make `query` optional and loop if there isn't one?

that seems like a pretty standard way to do things

Reply:
Well query is already optional, but the default behavior to loop is to be able to perform multiple searches without the expensive re-download. I could also just remove no-loop and force the user to ctrl-z, but that prevents the ability to pipe the output to another command (maybe I need a no-pager option)

Reply:
The idea of leaving loop as the default is that since this is such an expensive command, one can always peruse the results, with as many queries as they need to - so starting with no query may be expected.